db.py

```diff
@@ -97,4 +97,27 @@ def db_row_to_dict_task(row: tuple) -> dict:
         ]
     return dict(zip(column_names, list(row)))
 
+def get_user(conn, username) -> List[dict]:
```

Comment:
Well done, @phoebezhou22 !! The only thing I would change is updating `get_user` function to return a `dict` instead of a `List[dict]`. This is because in our `users` table, the `username` field has the `UNIQUE` constraint, meaning we know that if we find a user with the specified username in the table, that will be the one and only user wit that username! :)

How would you update the code to accomplish this?